documentation/tutorials/get-started.md

```diff
@@ -354,17 +357,19 @@ ticket
 
 ### Querying without persistence
 
-So far, there is no persistence happening. All that this simple resource does is return the record back to us. You can see this lack of persistence by attempting to use a `read` action:
+So far, we haven't stored any data in a database. All that this simple resource does is return the record back to us. You can see this lack of persistence by attempting to use a `read` action:
```

Comment:
We don't want to put across the idea that the only thing you might do with a resource is store it in a database. Might be worth making that distinction here. "Because we haven't used a data layer that does any persistence, for example storing records in a database."